svix-cli/src/cli_types/mod.rs

```diff
@@ -0,0 +1,41 @@
+//! Wrapper types around those in the SDK so they can act as args in the `clap` command tree.
+// FIXME: in the new codegen, we can look to add a `cli` feature that conditionally adds these
+//   attributes or trait impls to the wrapped types.
+//   At that point we can use the SDK types directly and remove this module.
```

Comment:
You want to keep this idea alive?

Reply:
I think what we're doing today is fine, but I do worry a bit about keeping the two sets of templates aligned.

Maybe we can branch in the template rather than doing a cargo feature (which was raised as objectionable)? I'll remove the FIXME regardless. I think we are aware of the situation by now.

Reply:
Right. Yeah we can use macros or includes in the templates to share code between CLI and lib codegen.